README.adoc

```diff
@@ -74,12 +58,10 @@ describe Article do
 end
 ----
 
-'''
+=== Empty line between example groups [[empty-lines-between-describes]]
```

Comment:
If you're going to assign a custom ID to a section, I recommend this syntax:

```
[#empty-lines-between-describes]
=== Empty lines between example groups
```

I also noticed that GitHub now acknowledges explicit IDs everywhere, so if you don't want to rely on the auto-generated ID, just use this syntax to assign a custom one.